linkerd/admin/src/main/scala/io/buoyant/linkerd/admin/names/WebDelegator.scala

```diff
@@ -168,7 +168,7 @@ private[admin] class WebDelegator(
   def apply(req: Request): Future[Response] = req.method match {
     case Method.Get => (req.params.get("d"), req.params.get("n")) match {
       case (Some(DtabStr(dtab)), Some(PathStr(path))) =>
-        val DstBindingFactory.Namer(namer) = linker.params[DstBindingFactory.Namer]
+        val DstBindingFactory.Namer(namer) = linker.routers.head.params[DstBindingFactory.Namer]
```

Comment:
Good question. It looks like all routers pick up the same set of name interpreters, here:

https://github.com/BuoyantIO/linkerd/blob/alex/config-madness/linkerd/core/src/main/scala/io/buoyant/linkerd/Linker.scala#L65

And since we require at least one router per config, then this assumption is probably ok, although I agree that it is confusing.